apps/expert/rel/remote.vm.args.eex

```diff
@@ -0,0 +1,13 @@
+## Customize flags given to the VM: https://www.erlang.org/doc/man/erl.html
+## -mode/-name/-sname/-setcookie are configured via env vars, do not set them here
+
+## Increase number of concurrent ports/sockets
+##+Q 65536
+
+## Tweak GC to run more often
+##-env ERL_FULLSWEEP_AFTER 10
+
+## Enable deployment without epmd
+## (requires changing both vm.args and remote.vm.args)
+-epmd_module Elixir.XPForge.EPMD
```

Comment:
Please double check this is indeed necessary. You may be fine using epmd for remote debugging (or maybe remote debugging is pointless).